WeakAuras/GenericTrigger.lua

```diff
@@ -1121,8 +1138,14 @@ function GenericTrigger.LoadDisplays(toLoad, loadEvent, ...)
   end
 
   for event in pairs(eventsToRegister) do
-    xpcall(frame.RegisterEvent, trueFunction, frame, event)
-    genericTriggerRegisteredEvents[event] = true;
+    local aceEventMessage = parseAceEventMessageType(event)
+    if aceEventMessage then
+      aceEvents:RegisterMessage(aceEventMessage, HandleMessage, frame)
+      genericTriggerRegisteredEvents[event] = true;  
```

Comment:
should I try to unregister messages somewhere? doesn't seem like that ever happens for the regular events.

Reply:
We don't track that in any place yet.